File: utils/VectorEnginePlugin.ref/pares/funciones/primInitializePath.js
function primInitializePath(argCount) {
    spanLeft = targetWidth;      // drawable right. Will later be refined.
    spanTop = targetHeight;      // drawable bottom. Will later be refined.
    spanRight = 0;               // drawable left. Will later be refined.
    spanBottom = 0;              // drawable top. Will later be refined.
    prevYRounded = 0x7FFFFFFF;
    return true;
}

File: utils/VectorEnginePlugin.ref/pares/funciones/primInitializePath.c
/* VectorEnginePlugin>>#initializePath */
EXPORT(sqInt)
primInitializePath(void)
{
	/* drawable right. Will later be refined. */
	spanLeft = targetWidth;

	/* drawable bottom. Will later be refined. */
	spanTop = targetHeight;

	/* drawable left. Will later be refined. */
	spanRight = 0;

	/* drawable top. Will later be refined. */
	spanBottom = 0;
	prevYRounded = 0x7FFFFFFF;
	return null;
}
